core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -248,6 +248,9 @@ static void *core_mmu_xlat_table_entry_pa2va(struct mmu_partition *prtn,
 					     unsigned int level,
 					     uint64_t entry);
 
+static bool core_mmu_entry_copy(struct core_mmu_table_info *tbl_info,
```

Comment:
We try to organize static functions in an order so we can avoid forward declarations.